client/settings/express-checkout-settings/woopay-settings.js

```diff
@@ -182,24 +182,27 @@ const WooPaySettings = ( { section } ) => {
 						<WooPayPreview
 							storeName={ wcSettings.siteTitle }
 							storeLogo={ woopayStoreLogo }
+							customMessage={ woopayCustomMessage }
 						></WooPayPreview>
-						<div className="woopay-settings__preview-fade"></div>
 					</div>
+					<hr
+						style={ {
+							margin: '0 -1.5rem 1.0rem -1.5rem',
+							width: 'calc(100% + 3rem)',
+						} }
+					/>
 					<div className="woopay-settings__custom-message-wrapper">
 						<h4>
-							{ __(
-								'Store logo on checkout',
-								'woocommerce-payments'
-							) }
+							{ __( 'Checkout logo', 'woocommerce-payments' ) }
 						</h4>
 						<WooPayFileUpload
 							fieldKey="woopay-store-logo"
 							accept="image/png, image/jpeg"
 							disabled={ false }
 							help={ __(
-								'Use a custom logo to WooPay if the one taken from your store doesn’t look right.' +
-									' For best results, upload a high-resolution horizontal image' +
-									' with white or transparent background.',
+								'Upload a custom logo. For best results, upload a horizontal' +
+									' image with white or transparent background. HEIC, WEBP, SVG, PNG,' +
+									' or JPG. Recommended width: 512 pixels minimum.',
```

Comment:
When I click to upload a logo, I can't select `.heic`, `.webp` or `.svg` formats. So we either need to include them in the `accept` property, or update the copy saying we only accept `.png` or `.jpg` images.

Reply:
Good catch. It looks like we plan on moving forward with only supporting `JPG` and `PNG` for now (https://github.com/Automattic/woocommerce-payments/pull/7511#issuecomment-1771899300). Updated the text copy in https://github.com/Automattic/woocommerce-payments/pull/7511/commits/28f508e049c3497b71ed765b5d04647395208405:
```
Upload a custom logo. For best results, upload a horizontal 
image with white or transparent background. PNG or JPG. 
Recommended width: 512 pixels minimum.
```